### ISSUE TITLE
Highlight evaluation

### ISSUE THREAD

**Issue by iwonac** [Cores-ts/Fundingbox-support#147]
Nice to have:
If you are an evaluator it would be nice to have an option to highlight some part of the application.

**Comment by xurxo** [comment]
Not sure what you mean. Like select and highlight a text?

**Comment by iwonac** [comment]
Yes, to highlight part of the application for further reading/comment/discussion.
Once you read the text you see the parts of the application you would like to discuss with others. As for now you have to copy part of the text somewhere - instead of this you could have a possibility to mark/highlight part of the text.
